.github/workflows/turborepo-release-step-3.yml

```diff
@@ -144,9 +144,6 @@ jobs:
           mv go-artifacts/turbo-go-cross-${{ inputs.release_branch }}/turbo_windows_amd64_v1/bin/* cli/dist-windows-amd64
           chmod a+x cli/dist-windows-amd64/turbo.exe
           chmod a+x cli/dist-windows-amd64/go-turbo.exe
-          mv go-artifacts/turbo-go-cross-${{ inputs.release_branch }}/turbo_windows_arm64/bin/* cli/dist-windows-arm64
```

Comment:
I think we want to continue shipping a windows/arm package, since we'll eventually be building it natively in Rust. So I think maybe this ought to be 
`mv go-artifacts/turbo-go-cross-${{ inputs.release_branch }}/turbo_windows_amd64_v1/bin/* cli/dist-windows-arm64`

And keep the following lines?